src/navigation/index.tsx

```diff
@@ -88,21 +88,23 @@ const NavigationRoot = () => {
               <>
                 <Stack.Screen name="TabNavigator" component={TabNavigator} />
                 <Stack.Screen
-                  name="TripDetailsModal"
-                  component={TripDetailsModal}
+                  name="TripDetailsRoot"
```

Comment:
I'm wondering if we could re-think a bit how these are used?

My thinking here is that if these two (`TripDetailsRoot`, `DepartureDetails`) are going to be sub-screens and not modals, they should be navigate-able within the context they are used in (`Assistant` and `Nearby`). In that case, `Nearby` and `Assistant` should probably define their own `Stack`-navigators which these two are used within. 

At the moment they are defined globally in the `NavigationRoot`, and we break certain patterns. They look like full-screen modals, but behave like sub-screens. 

In other words, I think we should show the tab-bar, and not lock the user into a full-screen context which is hard(er) to get out of.

Reply:
Sounds very reasonable! :raised_hands:

Reply:
In this way would we have to add specific stack.screen entries using the root here several places (e.g. departure, assistant, etc)

Reply:
My thinking was that Assistant gets its own Stack-Navigator. With two screen: the main `Assistant`-screen, and `TripDetailsRoot` as a screen. And Nearby gets two, the main one and  `DepartureDetails`. So specific screens per newly created `Navigator` yeah.

Reply:
Assistant has been provided its own stack navigator, and tripdetailsroot screen is moved to within Assistant navigator. :+1: 
DepartureDetails already exist as a screen in TripDetailsRoot stack navigator, and will remain there.
I am unsure whether to alter "Nearby" right now, or to do it in a separate branch after both mine Mikaels changes have been merged.